imagetest/test_suites/windows/driver_package_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build cit
+// +build cit
+
+package windows
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+func getDriverList(remove bool) []string {
+	drivers := []string{
+		"google-compute-engine-driver-netkvm",
+		"google-compute-engine-driver-pvpanic",
+		"google-compute-engine-driver-gga",
+		"google-compute-engine-driver-balloon",
+	}
+
+	// Do not remove Network or Disk
+	if !remove {
+		drivers = append(
+			drivers,
+			"google-compute-engine-driver-gvnic",
+			"google-compute-engine-driver-vioscsi",
+		)
+	}
+	return drivers
+}
+
+func getInstalledDrivers() ([]string, error) {
+	command := fmt.Sprintf("%s installed | Select-String google-compute-engine-driver-", googet)
+	output, err := utils.RunPowershellCmd(command)
+	if err != nil {
+		return nil, err
+	}
+
+	drivers := strings.Split(output.Stdout, "\n")
+
+	return drivers, nil
+
+}
+
+func TestNetworkDriverLoaded(t *testing.T) {
+	utils.WindowsOnly(t)
+	command := fmt.Sprintf("ipconfig /all | Select-String Description")
+	output, err := utils.RunPowershellCmd(command)
+	if err != nil {
+		t.Fatalf("Error running 'ipconfig': %v", err)
+	}
+	adapterName := "Google VirtIO Ethernet Adapter"
```

Comment:
This description is good for the virtio case.

GVNIC has a slightly different description. It's "Google Ethernet Adapter"

We'll likely need to test virtio and gvnic in different instances since an instance can only have one or another.